.github/workflows/go.yml

```
@@ -32,6 +32,11 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
     - uses: actions/checkout@v2
+      with:
+        # For pull requests, make sure we checkout `ref/pull/<PR_number>/head`
+        # and not `ref/pull/<PR_number>/merge` (which doesn't exist for the PR creator and
+        # is what github.ref points to)
```

Suggest giving a slightly less in the weeds description. When I read this I feel like it only makes sense to me because I have a lot of context already.
```suggestion
        # For pull requests, build and test the PR head not a merge of the PR with the destination.
```